### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by jwasinger**
Update the repo benchmarks with latest version and augment the makefile to run them with `make benchmark`

### CODE REVIEW

**Comment by cdetrio** [comment]
Something is up with `images/evm-vs-wasm-mul256-sol-rust.png`, the wasm engines aren't showing in the plot.

File: Makefile [hunk 1]
```diff
@@ -1,28 +1,41 @@
...
-build_docker_images:
-	cd evm/geth && docker build . -t geth-bench
-	cd evm/parity && docker build . -t parity-bench
-	cd evm/evmone && docker build . -t evmone-bench
-	cd evm/cita-vm && docker build . -t cita-vm-bench
-	cd evm/evmone-evm384 && docker build . -t evmone-evm384-bench
+all: build_docker_images benchmark notebook
+
+build_evm_engines:
+	cd evm/geth && docker build . -t ewasm/geth-bench:1
```
Comment (axic) [inline Makefile:4]: Do you want to create a PR which changes the makefile and scripts to use `ewasm/` ?

Comment (jwasinger) [reply Makefile:4]: is there a reason not to just include them here?  I don't see the scope of this PR expanding beyond what it currently is.

Comment (axic) [reply Makefile:4]: This PR already does multiple things:
1) changes a lot of the scripts/makefiles
2) runs new benchmarks and regenerates the notebook

Comment (jwasinger) [reply Makefile:4]: ok I'll split it out.

File: benchmark_results_data/backup-2020-06-25-1593057958/evm_benchmarks.csv [hunk 2]
```diff
@@ -0,0 +1,108 @@
+engine, test_name, total_time, gas_used
```
Comment (axic) [inline benchmark_results_data/backup-2020-06-25-1593057958/evm_benchmarks.csv:1]: Do we need these backup directories?

Comment (jwasinger) [reply benchmark_results_data/backup-2020-06-25-1593057958/evm_benchmarks.csv:1]: whoops.  no.  deleting those..

**Pull request merged**